stable/grafana/templates/deployment.yaml

```diff
@@ -129,7 +129,11 @@ spec:
         - name: storage
       {{- if .Values.persistence.enabled }}
           persistentVolumeClaim:
-            claimName: {{ .Values.persistence.existingClaim | default (include "grafana.fullname" .) }}
+          {{- if .Values.persistence.existingClaim -}}
```

Comment:
This would be a case when using the `default` function make sense. Sorry, I probably wasn't clear enough.

Reply:
@unguiculus  XD.  I was wondering, but was sticking with your `hard and fast` rule.  Sorry I took comment too literally.  Fixed